### ISSUE TITLE
Test Results Outside [0,1] Interval for LIQE

### ISSUE THREAD

**Issue by baoyu2020** [chaofengc/IQA-PyTorch#226]
Hello,
 I have encountered an issue with the test results for the LIQE model. The results are not within the expected [0,1] interval, which is different from the results obtained using the DBCNN model.
 
```
import torch
import cv2
import torch.nn.functional as F
import pyiqa

iqa_metric = pyiqa.create_metric("liqe_mix", device="cuda")
img = cv2.imread('/data1/home/Testcodec/bench/jpeg-ai/invcompress/q8/00003_TE_1944x1296.png')
img = torch.tensor(img).permute(2, 0, 1).float()
img = img.unsqueeze(0) / 255.0
print(img.shape)
print(iqa_metric(img))
del img
```

DBCNN Result: 0.49859
LIQE Result: 3.3864

Thank you for your attention

**Comment by chaofengc** [comment]
Please get the output score range with `metric.score_range`
